pkpdapp/pkpdapp/models/dataset.py

```diff
@@ -17,9 +19,10 @@ class Dataset(models.Model):
     ]
     name = models.CharField(max_length=100, help_text='name of the dataset')
     datetime = models.DateTimeField(
+        default=timezone.now,
         help_text=(
-            'Date/time the experiment was conducted. ',
-            'All time measurements are relative to this date/time',
+            'Date/time the experiment was conducted. '
+            'All time measurements are relative to this date/time'
         )
     )
     description = models.TextField(
```

Comment:
I also wonder whether description should be optional?

Reply:
yup, I'll set that to optional